gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.groovy

```diff
@@ -59,16 +59,29 @@ class JavaServiceDistributionPlugin implements Plugin<Project> {
 
         CreateStartScripts startScripts = CreateStartScriptsTask.createStartScriptsTask(project, 'createStartScripts')
         project.afterEvaluate {
-            CreateStartScriptsTask.configure(startScripts, distributionExtension.mainClass, distributionExtension.serviceName,
-                    distributionExtension.defaultJvmOpts, distributionExtension.enableManifestClasspath)
+            def jvmOpts = distributionExtension.defaultJvmOpts + (distributionExtension.heapSize != null
+                    ? ["-Xmx" + distributionExtension.heapSize, "-Xms" + distributionExtension.heapSize] : [])
```

Comment:
We usually want to pick these up from our config management system/would expect these to be parameterized by map keys. Maybe consider pushing further upstream into named parameters in go-java-launcher?

Reply:
Makes sense - will remove from this pr and figure out correct wiring